icu_benchmarks/models/alarm_metrics.py

```
@@ -0,0 +1,70 @@
+            continue
+        silence_array[positive_index + 1 : positive_index + silencing_length] = 0
+        positive_indices = positive_indices[1:]
+    # print(predictions)
+    # print(silence_array)
+    silenced_predictions = predictions * silence_array
+    # print(silenced_predictions)
+    return silenced_predictions
+
+
+def fill_gaps(predictions, ground_truth, grace_horizon=12):
+    """Fill gaps in the predictions by taking the maximum value between ground truth and predictions."""
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)